aws-cpp-sdk-core-tests/aws/config/AWSProfileConfigLoaderTest.cpp

```diff
@@ -36,6 +36,7 @@ static void WriteDefaultConfigFile(Aws::OStream& stream, bool useProfilePrefix =
 
     stream << "[ " << profilePrefix << "assumes_role]" << std::endl;
     stream << "role_arn =arn:aws:iam::123456789:role/foo" << std::endl;
+    stream << "external_id = +CsOo4kcN4UKViieKhF7qDs46qmM974XLOzp+TEs/9M=" << std::endl;
```

Comment:
I hope this is not a real key.
Please replace with some other text just in case.

Reply:
no, it's not a real key. This was generated using `openssl rand -base64 32`. I'll replace it with some other text, just for the looks of it.